src/main/java/com/hyperlocal/server/ShopController.java

```diff
@@ -59,25 +58,36 @@ public ShopController(PubSubTemplate pubSubTemplate) {
     connection = MySQLConnectionBuilder.createConnectionPool(Constants.DATABASE_URL);
   }
 
-  // API for performing text search
-  @GetMapping("/api/query/elastic")
-  public CompletableFuture<List<ShopDetails>> getDataFromElasticSearch(@RequestParam String query,
-      @RequestParam String queryRadius, @RequestParam String latitude, @RequestParam String longitude) {
+  // API for performing search and browse queries
+  @GetMapping("/v1/shops")
+  public CompletableFuture<List<ShopDetails>> getDataFromElasticSearch(
```

Comment:
getDataFromSearchIndex might be better name.

Reply:
Renamed it. Thanks a lot!!!